apps/bot_engine/lib/bot_engine/utils/utils.ex

```diff
@@ -0,0 +1,35 @@
+defmodule BotEngine.Utils.Utils do
+  def to_sentence(enum, options \\ %{})
+  def to_sentence(enum, _options) when length(enum) == 0, do: ''
+  def to_sentence(enum, _options) when length(enum) == 1 do
+    enum_to_string(Enum.at(enum, 0))
+  end
+
+  def to_sentence(enum, options) when length(enum) == 2 do
+    options = Map.merge(to_sentence_defaults, options)
+
+    "#{Enum.at(enum, 0)}#{options[:two_words_connector]}#{Enum.at(enum, 1)}"
+  end
+
+  def to_sentence(enum, options) when is_list(enum) do
+    options = Map.merge(to_sentence_defaults, options)
+
+    to_sentence_multiple_values(Enum.reverse(enum), options)
+  end
+
+
+  defp enum_to_string(entry) when is_binary(entry), do: entry
+  defp enum_to_string(entry), do: String.Chars.to_string(entry)
+
+  defp to_sentence_multiple_values([ head | tail ], options) do
+    "#{Enum.join(Enum.reverse(tail), options[:words_connector])}#{options[:last_word_connector]}#{head}"
```

Comment:
I had to interpolate `head` to a String first, because `"a" <> 1` raises an error, so

```elixir
Kernel.<>("#{head}")
```